src/app/Scenes/Home/Components/HeroUnitsRail.tsx

```diff
@@ -0,0 +1,121 @@
+import { Box, Flex, Spacer, Text, Button, Touchable } from "@artsy/palette-mobile"
+import {
+  HeroUnitsRail_heroUnitsConnection$data,
+  HeroUnitsRail_heroUnitsConnection$key,
+} from "__generated__/HeroUnitsRail_heroUnitsConnection.graphql"
+import OpaqueImageView from "app/Components/OpaqueImageView/OpaqueImageView"
+import { PaginationDots } from "app/Components/PaginationDots"
+import { navigate } from "app/system/navigation/navigate"
+import { extractNodes } from "app/utils/extractNodes"
+import { useScreenDimensions } from "app/utils/hooks"
+import { useRef, useState } from "react"
+import { FlatList, PixelRatio, ViewabilityConfig } from "react-native"
+import { useFragment } from "react-relay"
+import { graphql } from "relay-runtime"
+
+interface HeroUnitProps {
+  item: NonNullable<
+    NonNullable<NonNullable<HeroUnitsRail_heroUnitsConnection$data["edges"]>[number]>["node"]
+  >
+}
+
+const fontScale = PixelRatio.getFontScale()
+
+const CARD_HEIGHT = 250 * fontScale
+const CARD_IMAGE_WIDTH = 125
+const DESCRIPTION_LINES = fontScale > 1 ? 4 : 3
+
+const HeroUnit: React.FC<HeroUnitProps> = ({ item }) => {
+  const { width: screenWidth } = useScreenDimensions()
+  const cardImageWidth = screenWidth > 700 ? screenWidth / 2 : CARD_IMAGE_WIDTH
+
+  const handlePress = () => {
+    if (item.link.url) {
+      navigate(item.link.url)
+    }
+  }
+
+  return (
+    <Touchable key={item.internalID} onPress={handlePress}>
+      <Flex bg="black100" flexDirection="row" height={CARD_HEIGHT} width={screenWidth}>
+        <OpaqueImageView
+          height={CARD_HEIGHT}
+          imageURL={item.image?.imageURL}
+          resizeMode="cover"
+          width={cardImageWidth}
```

Comment:
The api of the new image is the same (already checked locally that it doesn't break the design) but it doesn't need the `resizeMode`

```suggestion
        <OpaqueImageView
          height={CARD_HEIGHT}
          imageURL={item.image?.imageURL}
          width={cardImageWidth}
```

Reply:
`import { Image } from '@artsy/palette-mobile'`